lazyfs/config/default.toml

```diff
@@ -10,5 +10,12 @@ blocks_per_page=1
 # page_size=4096
 # no_pages=10
 [filesystem]
-log_all_operations=false
-logfile=""
\ No newline at end of file
+log_all_operations=true
+logfile="/home/gsd/lfs.log"
+
+#[[injection]]
+#type="split_write"
+#file="/home/gsd/etcd-v3.4.25-linux-amd64/data-r/data.etcd/member/wal/0.tmp"
+#persist=[1]
+#parts=10
+#ocurrence=1
```

Comment:
If you intend to have predefined examples of fault configurations, can you create a folder here with those examples? I think it's better to maintain the main configuration file clean and simple.

Reply:
You're right. I am going to remove those lines from the default file and later I will add other examples of configurations in separate files!

Reply:
Also, is there some place where you explain how these faults work? I forgot to mention that, but if not please include in README.md a section for that.

Reply:
Added some info in the main README